src/confluent_kafka/src/Admin.c

```diff
@@ -2835,6 +3199,121 @@ static PyObject *Admin_c_DescribeConsumerGroupsResults_to_py(
+        result = PyObject_Call(ScramMechanism_type, args, value_dict);
+        Py_DECREF(value_dict);
+        Py_DECREF(ScramMechanism_type);
+        return result;
+}
+
+static PyObject *Admin_c_ScramCredentialInfo_to_py(const rd_kafka_ScramCredentialInfo_t *scram_credential_info){
+        PyObject *result = NULL;
+        PyObject *args = NULL;
+        PyObject *kwargs = NULL;
+        PyObject *ScramCredentialInfo_type = NULL;
+        rd_kafka_ScramMechanism_t mechanism;
+        int32_t iterations;
+
+        kwargs = PyDict_New();
+        mechanism = rd_kafka_ScramCredentialInfo_mechanism(scram_credential_info);
+        PyDict_SetItemString(kwargs,"mechanism",Admin_c_ScramMechanism_to_py(mechanism));
+
+        iterations = rd_kafka_ScramCredentialInfo_iterations(scram_credential_info);
+        cfl_PyDict_SetInt(kwargs,"iterations", iterations);
+        args = PyTuple_New(0);
+        ScramCredentialInfo_type = cfl_PyObject_lookup("confluent_kafka.admin",
+                                                       "ScramCredentialInfo");
+        result = PyObject_Call(ScramCredentialInfo_type, args, kwargs);
+        Py_DECREF(args);
+        Py_DECREF(kwargs);
+        Py_DECREF(ScramCredentialInfo_type);
+        return result;
+}
+
+static PyObject *Admin_c_UserScramCredentialsDescription_to_py(const rd_kafka_UserScramCredentialsDescription_t *description){
+        PyObject *result = NULL;
+        PyObject *args = NULL;
+        PyObject *kwargs = NULL;
+        PyObject *scram_credential_infos = NULL;
+        PyObject *UserScramCredentialsDescription_type = NULL;
+        int scram_credential_info_cnt;
+        int i;
+        kwargs = PyDict_New();
+        cfl_PyDict_SetString(kwargs, "user", rd_kafka_UserScramCredentialsDescription_user(description));
+
+        scram_credential_info_cnt = rd_kafka_UserScramCredentialsDescription_scramcredentialinfo_count(description);
+        scram_credential_infos = PyList_New(scram_credential_info_cnt);
+        for(i=0; i < scram_credential_info_cnt; i++){
+                const rd_kafka_ScramCredentialInfo_t *scram_credential_info =
+                        rd_kafka_UserScramCredentialsDescription_scramcredentialinfo(description,i);
+                PyList_SET_ITEM(scram_credential_infos,i,
+                        Admin_c_ScramCredentialInfo_to_py(
+                                scram_credential_info));
+        }
+
+        PyDict_SetItemString(kwargs,"scram_credential_infos",scram_credential_infos);
+        args = PyTuple_New(0);
+        UserScramCredentialsDescription_type = cfl_PyObject_lookup("confluent_kafka.admin",
+                                                                   "UserScramCredentialsDescription");
+        result = PyObject_Call(UserScramCredentialsDescription_type, args, kwargs);
+        Py_DECREF(args);
+        Py_DECREF(kwargs);
+        Py_DECREF(UserScramCredentialsDescription_type);
```

Comment:
Forgot to `Py_DECREF` for `scram_credential_infos `.

```suggestion
        Py_DECREF(args);
        Py_DECREF(kwargs);
        Py_DECREF(UserScramCredentialsDescription_type);
        Py_DECREF(scram_credential_infos );
```